tests/fixtures/dbt/sushi_test/schema.yaml

```diff
@@ -1,21 +1 @@
-- name: raw.items
```

Comment:
Why is this gone? Do we even need this file then?

Reply:
@crericha I believe this is from you (let me know if that isn't the case)

Reply:
The external model schema isn't needed, but can be useful for integration tests. We should bring this file back I think.

Reply:
added back